moderators.md

```diff
@@ -0,0 +1,137 @@
+
+# Joining
+
+This section describes the process to become a [moderator](governance.md#moderator).
+
+## Prerequisites
+
+[Moderators](governance.md#moderator) must be:
+
+1. Proficient in [R package development](https://r-pkgs.org/).
+2. Comfortable managing content on [GitHub](https://github.com) such as repositories, pull requests, issues, and discussions.
+3. Prepared to facilitate the [process to contribute packages to R-multiverse](https://r-multiverse.org/contributors.html).
+4. Prepared to facilitate discussions and answer questions on the [`r-multiverse/help`](https://github.com/r-multiverse/help)
+  [issue](https://github.com/r-multiverse/help/issues) and [discussion](https://github.com/r-multiverse/help/discussions) forums.
+5. Prepared to enforce [all R-multiverse policies](policies.md).
+
+## Selection
+
+[Administrators](governance.md#administrator) are responsible for selecting [moderators](governance.md#moderator).
+Beyond the [prerequisites](#prerequisites) above, [administrators](governance.md#administrator) choose [moderators](governance.md#moderator) based on trust.
+A [moderator](governance.md#moderator) should have a positive, strong, visible, verifiable presence in the R community.
+If you would like to be a [moderator](governance.md#moderator), please help [R-multiverse stakeholders](governance.md#r-multiverse-stakeholders) understand who you are and what you work on.
+There are countless ways to grow your public reputation, and each individual [moderator](governance.md#moderator) has their own unique type of impact.
+Here is just a small list of examples:
+
+* Contribute packages to well-known repositories (or R-multiverse).
+* Contribute or [review](https://ropensci.org/software-review/) for [rOpenSci](https://ropensci.org).
+* Organize or speak at an [R-focused conference](https://www.r-project.org/conferences/).
+* Organize a chapter of [R-Ladies](https://www.meetup.com/R-ladies/).
+* Curate [R Weekly](https://rweekly.org).
+* Contribute to an [R Consortium ISC working group](https://r-consortium.org/all-projects/isc-working-groups.html).
+* Contribute to an [ASA](https://community.amstat.org/) group such as [openstatsware](https://www.openstatsware.org).
+* Organize an [R user group](https://jumpingrivers.github.io/meetingsR/r-user-groups.html)
+* Publish R-focused articles in journals such as the [R Journal](https://journal.r-project.org), the [Journal of Statistical Software](https://www.jstatsoft.org), or the [Journal of Open Source Software](https://joss.theoj.org).
+
+## Volunteering
+
+If you would like to volunteer as a [moderator](governance.md#moderator), please fill out [this web form](). The form asks you about yourself:
+
+1. What is your full name?
+1. What is your professional affiliation?
+1. What is your GitHub username?
+1. Please list links that point to your online presence.
```

Comment:
Give some examples (social media, personal website, project website, blog, talks)

Reply:
Added in 32c07305195c7b6fa2303510a513ef90503391d4